docs/src/main/sphinx/connector/redshift.rst

```diff
@@ -109,6 +109,145 @@ that catalog name instead of ``redshift`` in the above examples.
 Type mapping
 ------------
 
+Because Trino and Redshift each support types that the other does not, this
+connector modifies some types when reading or writing data. Data types may not
+map the same way in both directions between Trino and the data source. Refer to
+the following sections for type mapping in each direction.
+
+Redshift type to Trino type mapping
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The connector maps Redshift types to the corresponding Trino types following
+this table:
+
+.. list-table:: Redshift type to Trino type mapping
+  :widths: 40, 40, 50
+  :header-rows: 1
+
+  * - Redshift database type
+    - Trino type
+    - Notes
+  * - ``BOOLEAN``
+    - ``BOOLEAN``
+    -
+  * - ``SMALLINT``
+    - ``SMALLINT``
+    -
+  * - ``INTEGER``
+    - ``INTEGER``
+    -
+  * - ``BIGINT``
+    - ``BIGINT``
+    -
+  * - ``REAL``
+    - ``REAL``
+    -
+  * - ``DOUBLE PRECISION``
+    - ``DOUBLE``
+    -
+  * - ``DECIMAL(p, s)``, ``NUMERIC(p, s)``
+    - ``DECIMAL(p, s)``
+    -
+  * - ``CHAR(n)``
+    - ``CHAR(n)``
+    -
+  * - ``VARCHAR(n)``
+    - ``VARCHAR(n)``
+    -
+  * - ``TEXT``
+    - ``VARCHAR(256)``
+    -
+  * - ``VARBYTE``
+    - ``VARBINARY``
+    -
+  * - ``DATE``
+    - ``DATE``
+    -
+  * - ``TIME(n)``
+    - ``TIME(n)``
+    -
+  * - ``TIMETZ(n)``
+    - ``TIME(n) WITH TIME ZONE``
+    -
+  * - ``TIMESTAMP(n)``
+    - ``TIMESTAMP(n)``
```

Comment:
This may be works, or maybe it doesn't. Sadly, we have no tests. We need to convince someone to donate a Redshift instance for Trino CI testing.